analysis.cxx

```diff
@@ -56,18 +56,24 @@ int main(){
     // Build the pair !
     auto df24_1 = physicsobject::FilterObjects(df24, "nTau", 1,"NumberOfTaus");
     auto df24_2 = physicsobject::FilterObjects(df24_1, "nMuon", 1,"NumberOfMuons");
-    auto df25 = pairselection::mutau::PairSelection(df24_2, "good_taus_mask", "good_muons_mask", "mtpair", {""});
-    auto df_final = df25;
+    auto mt_df = pairselection::mutau::PairSelection(df24_2, "good_taus_mask", "good_muons_mask", "mtpair", {""});
+    auto mt_df_1 = pairselection::filterGoodParticles(mt_df, "mtpair", "GoodMuTauPairs");
+    auto mt_df_2 = lorentzvectors::mutau::build(mt_df_1, "mtpair");
+
+
+    auto df_final = mt_df_2;
 
-    // Logger::get("main")->info(df_final.Describe()); <-- starting from ROOT 6.25
+    Logger::get("main")->debug(df_final.Describe()); // <-- starting from ROOT 6.25
 
     varSet.push_back("good_muons_mask");
     varSet.push_back("good_taus_mask");
     varSet.push_back("mtpair");
+    varSet.push_back("particle_1_p4");
+    varSet.push_back("particle_2_p4");
     Logger::get("main")->info("Finished Setup");
     Logger::get("main")->info("Starting Evaluation");
 
-    df_final.Snapshot<ROOT::VecOps::RVec<int>, ROOT::VecOps::RVec<int>, ROOT::VecOps::RVec<int>>("ntuple", "test.root", varSet);
```

Comment:
That's fine with me for the time-being.

However, as soon as we have a full list of quantities for the `mutau` pair to be written out, I'd suggest to test the non-jitted vs. jitted case.